airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlCdcStateHandler.java

```diff
@@ -51,4 +51,9 @@ public AirbyteMessage saveState(final Map<String, String> offset, final String d
     return new AirbyteMessage().withType(Type.STATE).withState(stateMessage);
   }
 
+  @Override
+  public AirbyteMessage saveStateAfterCompletionOfSnapshotOfNewStreams() {
+    throw new RuntimeException("Snapshot of individual tables currently not supported in MySQL");
```

Comment:
doublechecking: this only runs when someone adds new streams to an existing catalog, which isn't currently supported in the UI anyway?

Reply:
@edgao correct plus the stream state type for mysql is LEGACY so even when the feature would be enabled in the UI, it would not be triggered for mysql